README.adoc

```diff
@@ -55,6 +55,7 @@ brew cask install java
 cd $SPINNAKER_HOME
 mkdir build
 cd build
+mkdir deck
 ../spinnaker/dev/refresh_source.sh --pull_origin --use_ssh --github_user default
```

Comment:
Did you run `refresh_source.sh`? That will create that folder for you, among others. It downloads all the spinnaker services from git, and places them in their respective directories.